### PR TITLE
Add `termination_message_policy` parameter to `KubernetesPodOperator`

### PR DESCRIPTION
**Pull request by iJanki-gr**
Allow setting termination_message_policy in the Pod container.
My use case is being able to set the value of termination_message_policy to FallbackToLogsOnError. 
(https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html#include-error-message-in-email-alert)

From Pod specification:

terminationMessagePolicy (string)

Indicate how the termination message should be populated. File will use the contents of terminationMessagePath to populate the container status message on both success and failure. FallbackToLogsOnError will use the last chunk of container log output if the termination message file is empty and the container exited with an error. The log output is limited to 2048 bytes or 80 lines, whichever is smaller. Defaults to File. Cannot be updated.

### CODE REVIEW

File: airflow/providers/cncf/kubernetes/operators/pod.py [hunk 1]
```diff
@@ -238,6 +238,7 @@ class KubernetesPodOperator(BaseOperator):
         state, or the execution is interrupted. If True (default), delete the
         pod; if False, leave the pod.
         Deprecated - use `on_finish_action` instead.
+    :param termination_message_policy: set container termination_message_policy. Defaults to "File"
```
Comment (amoghrajesh) [inline airflow/providers/cncf/kubernetes/operators/pod.py]: Can you cover this behaviour using a unit test?

Comment (amoghrajesh) [inline airflow/providers/cncf/kubernetes/operators/pod.py]: nit: `sets container termination message policy. Default value is "File"`

Comment (iJanki-gr) [reply airflow/providers/cncf/kubernetes/operators/pod.py]: I added a test

**Comment by iJanki-gr** [comment]
Not sure if the failing test is a blip or not. How can I rerun the tests?

**Comment by amoghrajesh** [comment]
You can re run by closing and reopening the pull request or by rebasing the PR

File: airflow/providers/cncf/kubernetes/operators/pod.py [hunk 2]
```diff
@@ -238,6 +238,7 @@ class KubernetesPodOperator(BaseOperator):
         state, or the execution is interrupted. If True (default), delete the
         pod; if False, leave the pod.
         Deprecated - use `on_finish_action` instead.
+    :param termination_message_policy: sets container termination message policy. Default value is "File"
```
Comment (hussein-awala) [inline airflow/providers/cncf/kubernetes/operators/pod.py]: nit
```suggestion
    :param termination_message_policy: The termination message policy of the base container.
        Default value is "File"
```

**Review by hussein-awala — APPROVED** [review]
Looks good! LGTM

**Pull request merged**